### PR TITLE
Initial draft for a readme file

### PR DESCRIPTION
**Pull request by chaosmonster**
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,38 +9,138 @@ in markdown and compiled to the final page that is located in `docs`.
 
 ## CLI
```
Comment (kotzendekrabbe) [inline README.md:10]: IT's not a CLI yet

File: README.md [hunk 2]
```diff
@@ -9,38 +9,138 @@ in markdown and compiled to the final page that is located in `docs`.
...
+
+The `data` folder includes all changeable data for the Sinnerschrader career page.
+It uses *markdown* and *front matter* to structure the information. Data is divided in `jobs` and `contact-persons`.
+
 ### Markdowns
 
-Was ist frontmatters
+#### What is Markdown
+
+TODO: find a good tutorial or explanation
```
Comment (kotzendekrabbe) [inline README.md:33]: could we create an issue for it? maybe an issue with all open documentation, but that would be awesome :)

File: README.md [hunk 3]
```diff
@@ -9,38 +9,138 @@ in markdown and compiled to the final page that is located in `docs`.
...
+|language | `en`/ `de`| defines the language for the job post| yes |
+|location | `Ber`/ `Fra`/ `Ham`/ `Muc` / `Pra` | Short key for the location the job will be located at | yes |
+|discipline | `Account`/ `Creative`/ `Strategy`/ `Tech`| This key is to provide the information for the discipline | yes |
+|level | `Internships`/ `Professionals`/ `Students`/ `Young Professionals` | What level of professionality is searched for | yes |
+|company | `SinnerSchrader`/ `Swipe`/ `Commerce` | TODO: do we need this? | maybe |
+|date | example `2020-10-21` | The date this job post was published (it will not be auto published) the format is year - month (always two digits) - day (always to digits)| maybe |
+|published | `true`/ `false` | This flag defines if the job post will be published or not | yes |
+|title | Text | The title shown for this job post, it will also be used for social sharing | yes |
+|description | Text | The description will only be used for social sharing | no |
+|contact | `contact key` | This is the file name of the contact data that will be shown. e.g. 'victoria'. If not provided the 'default' contact will be used| no |
```
Comment (kotzendekrabbe) [inline README.md:60]: please get in contact with victoria, robert and annika who to use here

**Review by kotzendekrabbe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chaosmonster) [reply README.md:10]: see #10 - will be fixed there

File: README.md (hunk 3 above)
Comment (chaosmonster) [reply README.md:60]: will rename it to martin

File: README.md (hunk 2 above)
Comment (chaosmonster) [reply README.md:33]: Added issue for this and the following section. see #11

**Pull request merged**